### PR TITLE
fix(vesting+shell): render fixes

### PR DESCRIPTION
**Pull request by olegshilov**
- remove `Transition` component usage
- use data attrs instead render function arg to style some account and select chain button states

### CODE REVIEW

File: libs/ui-kit/src/lib/sort-select.tsx [hunk 1]
```diff
@@ -85,25 +85,15 @@ export function SortSelect({
...
-        enterFrom="opacity-0 scale-95"
-        enterTo="opacity-100 scale-100"
-        leave="ease-in duration-75"
-        leaveFrom="opacity-100 scale-100"
-        leaveTo="opacity-0 scale-95"
+      <MenuItems
+        className={clsx(
+          'absolute left-[0px] z-10 mt-[4px] w-fit min-w-full origin-top rounded-[8px] bg-[#0D0D0E] py-[8px] shadow-lg focus:outline-none',
+          'max-h-[256px] overflow-y-scroll',
+          'border-haqq-border border-[1px]',
```
Comment (viktoriabakun) [inline libs/ui-kit/src/lib/sort-select.tsx]: just wanted to ask if there's any need in writing like that: `border-[1px]` , `left-[0px],` 
when you can write like that: `border` , `left-0`

Screenshot 2024-08-26 at 12 28 52

Screenshot 2024-08-26 at 12 28 04

**Review by viktoriabakun — APPROVED** [review]

File: libs/ui-kit/src/lib/account-button.tsx [hunk 2]
```diff
@@ -31,61 +30,46 @@ export function AccountButton({
...
-                as="button"
-                className="block w-full px-[16px] py-[10px] text-left text-[13px] leading-[22px] transition-colors duration-150 ease-out hover:bg-[#ffffff14]"
-                onClick={onDisconnectClick}
-              >
-                Disconnect
-              </Menu.Item>
-            </Menu.Items>
-          </Transition>
+            <div className="mt-[-1px]">{address}</div>
+            <svg
```
Comment (GarageInc) [inline libs/ui-kit/src/lib/account-button.tsx]: SVG убрал бы в отдельный компонент

File: libs/ui-kit/src/lib/select-chain-button.tsx [hunk 3]
```diff
@@ -19,116 +18,96 @@ export function SelectChainButton({
...
+            'flex h-[40px] flex-row items-center rounded-[6px] px-[12px]',
+            'font-guise text-[14px] font-[500] leading-[22px]',
+            'transition-colors duration-150 ease-out',
+            'box-border appearance-none outline-none',
+            'hover:text-haqq-black border border-white text-white hover:bg-white',
+            'data-[open]:!text-haqq-black data-[open]:!bg-white',
+          )}
+        >
+          <div className="mt-[-1px]">{currentChain.name}</div>
+          <svg
```
Comment (GarageInc) [inline libs/ui-kit/src/lib/select-chain-button.tsx:34]: same

**Review by GarageInc — CHANGES_REQUESTED** [review]

File: libs/ui-kit/src/lib/sort-select.tsx (hunk 1 above)
Comment (olegshilov) [reply libs/ui-kit/src/lib/sort-select.tsx]: actually it doesn't matter cuz it generate same rules in result css.
but this values dosen't rely on tailwind configuration.

**Review by GarageInc — APPROVED** [review]

**Pull request merged**